### PR TITLE
Layout for the application

### PR DESCRIPTION
**Pull request by artur-gajewski**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import Logo from './Logo';
+
+const Wrapper = styled.div`
+  background-color: ${props => props.theme.colors.blue};
+  color: #fff;
+  padding-top: 4em;
+  height: 15em;
+`;
+
+const OptionsContainer = styled(Container)`
```
Comment (Cifixie) [inline src/components/Footer.js]: There's an option `fluid` which can be passed to container which acts as full-width

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  width: 100%;
+`;
+
+const LinksContainer = styled(Container)`
+  margin-top: 2em;
+  width: 60%;
```
Comment (Cifixie) [inline src/components/Footer.js]: Fine as it's now, later we can change this to "max width" or similar.

File: src/components/KoroSection.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+@import '../../node_modules/open-city-design/src/scss/helsinki/mixins';
+
+.section-koro-top {
+  &.fog:after {
+    @include koro('basic', #9fc8eb, 400);
```
Comment (Cifixie) [inline src/components/KoroSection.scss]: We can get the Colors from _colors... If we stick with this method, something we need to change later on.

File: src/components/Legend.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import styled from 'styled-components';
+import { Col, Row, Container } from 'reactstrap';
+
+const Legend = styled.div`
+  background-color: ${props => props.theme.colors.helFog};
+  padding-top: 3em;
+`;
+
+const LegendContainer = styled(Container)`
+  width: 80%;
```
Comment (Cifixie) [inline src/components/Legend.js]: Fine as it's now, later we can change this to "max width" or similar.

File: src/components/Logo.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import styled from 'styled-components';
+import logo from 'open-city-design/src/assets/helsinki-logo-black.svg';
```
Comment (Cifixie) [inline src/components/Logo.js]: We already talked in the morning, that at some point we should switch into: https://github.com/airbnb/babel-plugin-inline-react-svg to have better control over the SVG's

File: src/components/Page.js [hunk 6]
```diff
@@ -1,14 +1,71 @@
...
+  background-color: white;
+  padding-bottom: 3.125rem;
+`;
+
+type Props = {
+  children: any
+};
+
+const TopNavbar = styled(Navbar)`
+  background-color: ${props => props.theme.colors.helFog};
```
Comment (Cifixie) [inline src/components/Page.js]: 👍

File: src/components/Steps.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  height: 3.125em;
+  display: inline-block;
+  border: 0.125em solid #000;
+  border-radius: 50%;
+  padding-top: 0.7em;
+  font-size: 1.2em;
+  color: #fff;
+  margin-bottom: 1em;
+
+  &.active {
```
Comment (Cifixie) [inline src/components/Steps.js]: Prefer `props` over classNames, that's why styled-component is for

Comment (artur-gajewski) [reply src/components/Steps.js]: Yes, this will be refactored next when each step will be its own component.

**Pull request merged**